packages/components/src/Spinner/src/Spinner.tsx

```diff
@@ -59,15 +60,18 @@ const Spinner = (props: SpinnerProps, ref: ForwardedRef<HTMLDivElement>) => {
         ...stylingProps.style,
         ...style
     };
+    
+    const labelId = useId();
 
     const labelMarkup = children && (
-        <Label
+        <Text
```

Comment:
What is the reason for this change?

Reply:
because it's not really a label. The label component is used with field components or groups. We used Text inside the Checkbox and Radio components as well.

Unless you think this can be done differently?
The font that the Label was using before was for the body. but in figma, it's supposed to use Heading xs. I believed that the Heading xs font was the true label.

Reply:
that's fine, I was wondering about the reason for the change. Thanks for the clarification.